test/methods.js

```diff
@@ -1282,4 +1314,25 @@ test("cpfBR", function() {
 	ok( !method( "11144477737"), "Invalid CPF Number: 2nd check number failed");
 });
 
+test("file accept - image wildcard", function() {
+	var input = acceptFileDummyInput("test.png", "image/png"),
+			$form = $("<form />"),
+			proxy = $.proxy($.validator.methods.accept, new $.validator({}, $form[0]), null, input, "image/*");
+	ok( proxy(), "the selected file for upload is valid" );
+});
+
+test("file accept - specified mime type", function() {
+	var input = acceptFileDummyInput("test.kml", "application/vnd.google-earth.kml+xml"),
+			$form = $("<form />"),
+			proxy = $.proxy($.validator.methods.accept, new $.validator({}, $form[0]), null, input, "application/vnd.google-earth.kml+xml");
+	ok( proxy(), "the selected file for upload is valid" );
+});
+
+test("file accept - invalid mime type", function() {
+	var input = acceptFileDummyInput("test.kml", "foobar/vnd.google-earth.kml+xml"),
+			$form = $("<form />"),
+			proxy = $.proxy($.validator.methods.accept, new $.validator({}, $form[0]), null, input, "application/vnd.google-earth.kml+xml");
+	equal(proxy(), "dependency-mismatch", "the selected file for upload is valid" );
```

Comment:
The expected value doesn't match the test name ("invalid mime type"), I'd expect a `false` here (in that case, using `strictEqual` would also help).

It also looks like the message argument was just copied from previous assertions and doesn't match here.